styles/colors/colors.scss

```diff
@@ -1,6 +1,7 @@
 // Backgrounds
 $background-color: #f8f1f1;
 $form-background: #fdfeff;
+$header-background: #fff;
```

Comment:
could you let me know why same #fff has different variables ?
could you let me know are #fff & #ffffff are same ?

Reply:
Both represent the same, but #fff is a shorthand notation and #ffffff is a longhand notation. 

Shorthand notation = #RGB , Longhand notation = #RRGGBB

Reply:
thank you for explaining, so in an application, could you let me know,  is it a good practice to have the same colour in different notation?

Reply:
In terms of consistency, yes in this case it is better to have shorthand notation in all places.